### ISSUE TITLE
Looks like the app errors on startup?

### ISSUE THREAD

**Issue by fluffywaffles** [heroku/clojure-getting-started#3]
See https://cryptic-falls-9493.herokuapp.com/#

It gives a parse error on load

**Comment by olegakbarov** [comment]
It does.

**Comment by jonmountjoy** [comment]
I can't reproduce this.  If I clone the repo, run `heroku create` and then `git push heroku master` and then open the app, I see the expected `["Hello" :from Heroku]`.

Are you doing anything differently?

**Comment by jkutner** [comment]
Does the error you get look something like this?

```
Error: Parse error on line 1:
["Hello" :from Heroku]
---------^
Expecting 'EOF', '}', ',', ']'
```

If so, I haven't been able to reproduce, but there have been a few other folks reporting this. It looks like something is trying to parse the output as JSON, even though it's not. Do you know what that could be? What browser are you testing this in? Can you try others (Firefox at least?).

**Comment by skorlir** [comment]
That looks right, so far as the error message, but I just redeployed and I'm not even getting that - just a generic Heroku Application Error. I'm going to try starting over fresh and deploying a new one, to see if the error comes back

**Comment by jkutner** [comment]
@skorlir try running `heroku ps:scale web=1`. I suspect your dyno was scaled down.

**Comment by skorlir** [comment]
@jkutner I did. It could be that I modified the starter code somehow at some point.

**Comment by jkutner** [comment]
@skorlir can you run `heroku logs`, and look to see if there is a clojure specific error? the generic Application Error is just Heroku returning a 500

**Comment by skorlir** [comment]
Here's the log output:

```
/usr/local/heroku/lib/heroku/jsplugin.rb:87: warning: Insecure world writable dir /home/jordan/dev/classes in PATH, mode 040777
2016-01-06T09:48:28.691838+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=cryptic-falls-9493.herokuapp.com request_id=ad2b0267-2aed-4366-bd9a-2d4fa4cf01d2 fwd="81.31.127.166" dyno= connect= service= status=503 bytes=
2016-01-06T09:48:30.966746+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=cryptic-falls-9493.herokuapp.com request_id=f0b43e55-ebc3-404b-a3f8-09e4d9ce7e41 fwd="81.31.127.166" dyno= connect= service= status=503 bytes=
2016-01-06T09:48:34.416239+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=cryptic-falls-9493.herokuapp.com request_id=941e57cf-4d37-4475-9228-ecc820cd1b52 fwd="81.31.127.166" dyno= connect= service= status=503 bytes=
2016-01-06T09:48:35.374172+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=cryptic-falls-9493.herokuapp.com request_id=d41e20a1-700a-40a8-a22f-3363a34b5636 fwd="81.31.127.166" dyno= connect= service= status=503 bytes=
2016-01-06T09:48:45.363014+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=cryptic-falls-9493.herokuapp.com request_id=ce1c9eac-c731-41d6-b70f-dd1a20e0f883 fwd="81.31.127.166" dyno= connect= service= status=503 bytes=
2016-01-06T09:48:46.189644+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=cryptic-falls-9493.herokuapp.com request_id=8aeb9aac-a2d9-49a9-9a79-d873a0935fab fwd="81.31.127.166" dyno= connect= service= status=503 bytes=
2016-01-06T09:48:47.527599+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=cryptic-falls-9493.herokuapp.com request_id=f8a637b5-f45e-40db-85ee-159890be6e71 fwd="81.31.127.166" dyno= connect= service= status=503 bytes=
2016-01-06T09:48:48.158518+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=cryptic-falls-9493.herokuapp.com request_id=9f114d92-8e70-4da1-9a58-09920d64877f fwd="81.31.127.166" dyno= connect= service= status=503 bytes=
2016-01-06T15:27:06.789747+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=cryptic-falls-9493.herokuapp.com request_id=713ad2d2-d4b1-45d1-9b9e-e55c382edd32 fwd="12.19.147.253" dyno= connect= service= status=503 bytes=
2016-01-06T23:05:15.920386+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=cryptic-falls-9493.herokuapp.com request_id=5f787114-9ba3-4252-9472-09891c6385f3 fwd="165.124.144.171" dyno= connect= service= status=503 bytes=
2016-01-06T23:05:16.407896+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=cryptic-falls-9493.herokuapp.com request_id=68f06727-62f7-489e-9886-78817f4f9980 fwd="165.124.144.171" dyno= connect= service= status=503 bytes=
2016-01-06T23:05:19.344352+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=cryptic-falls-9493.herokuapp.com request_id=76a0797d-34b0-42c9-842b-c6a85a54701e fwd="165.124.144.171" dyno= connect= service= status=503 bytes=
2016-01-06T23:05:19.604082+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=cryptic-falls-9493.herokuapp.com request_id=5074cd9f-bda3-46cf-b216-161c8ea176af fwd="165.124.144.171" dyno= connect= service= status=503 bytes=
2016-01-06T23:05:20.016982+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=cryptic-falls-9493.herokuapp.com request_id=c8de0864-55e3-4eca-93ab-45b88dd6b6dc fwd="165.124.144.171" dyno= connect= service= status=503 bytes=
2016-01-06T23:05:20.257421+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=cryptic-falls-9493.herokuapp.com request_id=79985a99-f808-4df3-8efe-bfee3f02963c fwd="165.124.144.171" dyno= connect= service= status=503 bytes=
2016-01-06T23:05:20.509679+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=cryptic-falls-9493.herokuapp.com request_id=969d9f15-c2e4-4cc9-8b58-d0019138a908 fwd="165.124.144.171" dyno= connect= service= status=503 bytes=
2016-01-06T23:05:20.723307+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=cryptic-falls-9493.herokuapp.com request_id=a9bc1a62-7a50-464e-b673-2acaf0638490 fwd="165.124.144.171" dyno= connect= service= status=503 bytes=
2016-01-06T23:05:26.260721+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=cryptic-falls-9493.herokuapp.com request_id=dbd7e912-9b61-4f00-8d09-69155cee8beb fwd="165.124.144.171" dyno= connect= service= status=503 bytes=
2016-01-06T23:05:26.643583+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=cryptic-falls-9493.herokuapp.com request_id=744d7174-86f3-4ce9-9335-7e5cf457a30a fwd="165.124.144.171" dyno= connect= service= status=503 bytes=
2016-01-06T23:15:23.852156+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=cryptic-falls-9493.herokuapp.com request_id=4cadeb20-7eec-4682-b300-47597d64983d fwd="165.124.144.171" dyno= connect= service= status=503 bytes=
2016-01-06T23:15:24.206411+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=cryptic-falls-9493.herokuapp.com request_id=15217712-92e4-40bf-a982-17c090bb05ce fwd="165.124.144.171" dyno= connect= service= status=503 bytes=
2016-01-06T23:16:26.916482+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=cryptic-falls-9493.herokuapp.com request_id=aec6cdd8-38d1-4d5d-8559-12db27c3d436 fwd="165.124.144.171" dyno= connect= service= status=503 bytes=
2016-01-06T23:16:27.372245+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=cryptic-falls-9493.herokuapp.com request_id=2e9ab2cb-8aa6-4e55-aa83-22d6d3850e06 fwd="165.124.144.171" dyno= connect= service= status=503 bytes=
2016-01-06T23:16:27.328355+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=cryptic-falls-9493.herokuapp.com request_id=513f3311-8788-40a4-a8f4-d54411ab5680 fwd="165.124.144.171" dyno= connect= service= status=503 bytes=
2016-01-06T23:16:27.755540+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=cryptic-falls-9493.herokuapp.com request_id=e2f46022-753c-40a0-ac30-9d0e5696c814 fwd="165.124.144.171" dyno= connect= service= status=503 bytes=
2016-01-06T23:16:42.216723+00:00 heroku[router]: at=error code=H99 desc="Platform error" method=GET path="/" host=cryptic-falls-9493.herokuapp.com request_id=0079fc73-46d3-4fe1-a75e-15c55816bbfd fwd="165.124.144.171" dyno= connect= service= status=503 bytes=
2016-01-06T23:22:58.931270+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=cryptic-falls-9493.herokuapp.com request_id=5677a101-bd6c-43cf-b74c-44bbf0a509ae fwd="165.124.144.171" dyno= connect= service= status=503 bytes=
2016-01-06T23:22:59.286319+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=cryptic-falls-9493.herokuapp.com request_id=72046d09-ed82-4ca7-bcf6-d0c4dd5b964d fwd="165.124.144.171" dyno= connect= service= status=503 bytes=
```

**Comment by jkutner** [comment]
@skorlir that looks like a [boot timeout](https://devcenter.heroku.com/articles/error-codes#h10-app-crashed). Probably related to code changes. Can you open a [support ticket](http://help.heroku.com) so we can take a closer look (and move the discussion of of this thread)? thanks.

**Comment by MaxmumPimp** [comment]
@jkutner - I got the same error- it's strange because it even adds a little triangular warning icon before the keyword:
<img width="862" alt="screen shot 2016-02-25 at 2 35 09 pm" src="https://cloud.githubusercontent.com/assets/5484828/13333505/ffd665e8-dbcc-11e5-8f9b-4408c53c412a.png">

I think this is Chrome's problem, it's trying to be too smart and to parse the content as a JSON Array. In FF, you can see it plainly and clearly. Viewing source just shows exactly the content you'd expect.

**Comment by jkutner** [comment]
Fixed by 1133b880478d6f11e14250e245ce521e478fb9e4

Thank you @MaxmumPimp
